src/ui/screens/ResultDetails.tsx

```diff
@@ -18,6 +18,26 @@ const itemList: any = {};
 
 export default class ResultDetails extends React.Component<DetailsProps> {
 
+    getPIIDisplayString(key:string, val:string) {
+        try {
+            let piiString = val;
+            const dataKey = _.findKey(CredentialKeys, (item) => {return item.name === key});
+            if (dataKey && CredentialKeys[dataKey] && CredentialKeys[dataKey].dataType === "date") {
+                piiString = new Date(Number(val)).toLocaleDateString(
+                    'en-gb',
```

Comment:
Unless there is an specific reason for using the Great Britain locale region we should have en-US for now.
Not really a blocker for this PR though.